draft-irtf-cfrg-opaque.md

```diff
@@ -1415,11 +1400,12 @@ would use only the ephemeral key shares exchanged between client and server.
 # Test Vectors
 
 This section contains test vectors for the OPAQUE-3DH specification. Each test
-vector specifies the configuration information, protocol inputs, intermeidate
+vector specifies the configuration information, protocol inputs, intermediate
 values computed during registration and authentication, and protocol outputs.
 All values are encoded in hexadecimal strings. The configuration information
 includes the (OPRF, Hash, MHF, EnvelopeMode, Group) tuple, where the Group
-matches that which is used in the OPRF.
+matches that which is used in the OPRF. These test vectors were generated using
+draft-06 of {{I-D.irtf-cfrg-voprf}}.
```

Comment:
> draft-06

In the future, it might become difficult to remember to update this number, right? Should we just eliminate this sentence?

Reply:
I left it here to make it clear that we’re using -06. I’m not worried about updating this in the future.